rest-api-json-bulk/src/main/kotlin/BulkAccessWrapper.kt

```diff
@@ -0,0 +1,51 @@
+import University.Schedule.api.gen.University_Schedule
+import University.Schedule.api.gen.jetbrains_mps_lang_core
+import org.modelix.model.api.INode
+import org.modelix.model.api.INodeReference
+import org.modelix.model.lazy.INodeReferenceSerializer
+import org.modelix.mps.apigen.runtime.MPSLanguageRegistry
+import org.modelix.mps.rest.model.access.api.ModelView
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+val logger: Logger = LoggerFactory.getLogger("BulkAccessWrapper")
+
+object BulkAccessWrapper {
+
+    private lateinit var client: MPSRemoteClient
+    private lateinit var modelsToLoad: List<ModelView>
+
+    suspend fun initialize(host: String, port: Int, models: List<String>){
+        registerLanguages()
+
+        client = MPSRemoteClient(host, port)
+        modelsToLoad = client.getViewModels().filter { models.contains(it.name) }
+
+        if (models.size != modelsToLoad.size) {
+            logger.error("Not all models available!")
+            logger.error(
+                    """Missing models: ${
+                        models.filterNot { modelsToLoad.map { it.name }.contains(it) }.joinToString()
+                    }"""
+            )
+            return
```

Comment:
Maybe an exception would be better. When this condition occurs, the serializer isn't registered and therefore things would be unusable in the end.